packages/dds/sequence/api-report/sequence.api.md

```diff
@@ -325,6 +325,14 @@ export enum IntervalType {
 
 // @public (undocumented)
 export interface IOverlappingIntervalsIndex<TInterval extends ISerializableInterval> extends IntervalIndex<TInterval> {
+    // (undocumented)
```

Comment:
Can we add doc comments to these new methods?

Reply:
Yes, will add it in the following commits